mirage/src/main/resources/application.properties

```diff
@@ -15,6 +15,10 @@ management.metrics.export.influx.password=user
 management.metrics.export.influx.step=PT5s
 management.metrics.export.influx.uri=http://localhost:8086
 
+management.metrics.distribution.percentiles.all=.75,.90,.95,.99,.999
+# percentile histograms only work for Prometheus (https://micrometer.io/docs/concepts#_histograms_and_percentiles)
+management.metrics.distribution.percentiles-histogram.all=true
```

Comment:
This will yield way too many buckets. We should rather go with some predefined ones with e.g.:
```
management.metrics.distribution.slo=100ms,200ms,400ms
```